plugins/wazuh_check_updates/public/components/dismissNotificationCheck.tsx

```diff
@@ -0,0 +1,39 @@
+import { EuiCheckbox } from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage } from '@osd/i18n/react';
+import { useUserPreferences } from '../hooks';
+
+export const DismissNotificationCheck = () => {
+  const [dismissFutureUpdates, setDismissFutureUpdates] = useState<boolean>();
+
+  const { userPreferences, error, isLoading, updateUserPreferences } = useUserPreferences();
+
+  useEffect(() => {
+    if (isLoading) return;
+    setDismissFutureUpdates(userPreferences?.hide_update_notifications);
+  }, [userPreferences, isLoading]);
+
+  if (error) {
+    console.log(error);
```

Comment:
**nitpick**: we should decide how to log the errors.

Reply:
console.log deleted

Reply:
console.log deleted